packages/ui/src/hooks/useErrorMessage.ts

```diff
@@ -244,6 +247,10 @@ export const useErrorMessage = () => {
           'Core has entered an unexpected state. Please restart the browser if the issue persists.',
         ),
       },
+      [CommonError.InvalidAmount]: {
```

Comment:
Is there no need to use this hook anywhere? You added some error codes here, but the hook itself doesn't seem to be used.